src/Nethermind/Nethermind.Trie/TrieNode.Visitor.cs

```diff
@@ -30,13 +30,14 @@ public partial class TrieNode
         /// <param name="nextToVisit"></param>
         /// <exception cref="InvalidDataException"></exception>
         /// <exception cref="TrieException"></exception>
-        internal void AcceptResolvedNode(ITreeVisitor visitor, ITrieNodeResolver nodeResolver, SmallTrieVisitContext trieVisitContext, IList<(TrieNode, SmallTrieVisitContext)> nextToVisit)
+        internal void AcceptResolvedNode<TNodeContext>(ITreeVisitor<TNodeContext> visitor, in TNodeContext nodeContext, ITrieNodeResolver nodeResolver, SmallTrieVisitContext trieVisitContext, IList<(TrieNode, TNodeContext, SmallTrieVisitContext)> nextToVisit)
```

Comment:
Why is struct `in` here but not other places?

Reply:
I haven't found any other places where it's not `in`, beside local copies.

Reply:
@benaadams ☝️